src/services/print.js

```diff
@@ -488,6 +492,63 @@ ngeo.Print.prototype.encodeVectorStyleStroke_ =
 
 
 /**
+ * @param {Array.<MapFishPrintSymbolizer>} symbolizers Array of MapFish Print
+ *     symbolizers.
+ * @param {!ol.style.Text} textStyle Text style.
+ * @private
+ */
+ngeo.Print.prototype.encodeTextStyle_ = function(symbolizers, textStyle) {
+  var symbolizer = /** @type {MapFishPrintSymbolizerText} */ ({
```

Comment:
It's not really a constructor. It is a constructor because it is the way types are defined in an externs file.